torch/testing/_internal/common_methods_invocations.py

```diff
@@ -782,8 +784,8 @@ def _generate_reduction_inputs(device, dtype, requires_grad):
     """Generates input tensors for testing reduction operators"""
     yield make_tensor([], device, dtype, requires_grad=requires_grad)
     yield make_tensor([2], device, dtype, requires_grad=requires_grad)
-    yield make_tensor([2, 3], device, dtype, requires_grad=requires_grad, noncontiguous=True)
-    yield make_tensor([3, 2, 1, 5], device, dtype, requires_grad=requires_grad)
+    yield make_tensor([3, 5], device, dtype, requires_grad=requires_grad, noncontiguous=True)
```

Comment:
Why the change of size?

Reply:
To keep the test cases small and prevent overflow with prod.

Reply:
You go from 2, 3 to 3, 5. That doesn't really make the test case smaller? :D

Reply:
I made the [3, 2, 1, 5] -> [3, 2, 1, 2] smaller and [2, 3] -> [3, 5] bigger to compensate? Haha, these are arbitrary numbers at the end of the day.

Reply:
Yes they are. That's why I was surprised you changed them.